pkg/sshutil/sshutil.go

```diff
@@ -162,3 +150,25 @@ func SSHArgs(instDir string) ([]string, error) {
 	)
 	return args, nil
 }
+
+func SSHArgs(instDir string) ([]string, error) {
+	controlSock := filepath.Join(instDir, filenames.SSHSock)
+	if len(controlSock) >= osutil.UnixPathMax {
+		return nil, errors.Errorf("socket path %q is too long: >= UNIX_PATH_MAX=%d", controlSock, osutil.UnixPathMax)
+	}
+	u, err := user.Current()
+	if err != nil {
+		return nil, err
+	}
+	args, err := SCPArgs()
```

Comment:
This is confusing, can we just use a single function like `func SSHArgs(instDir string, useControlMaster bool)` ?

Reply:
> This is confusing,

Not sure why this is confusing: the SCP arguments are just a subset of the SSH arguments. There are also potential differences (not relevant here), like `-p` for  SSH is `-P` for SCP because `-p` for SCP means "preserve access mode and file times"...

> can we just use a single function like func SSHArgs(instDir string, useControlMaster bool) ?

So you would call it with an empty string as the `instDir` from the `copyCommand`? Because we no longer have a global `instDir`, it can be different for each source argument.

Please reconsider! I think the current implementation is less confusing than forcing both commands to use the same function.

Reply:
I was confused because "SCPArgs" is called even when we do not use scp.

Just renaming `SCPArgs()` to `CommonArgs()` and let `var SCPArgs = CommonArgs` might be less confusing, but anyway I don't have a strong opinion.

Reply:
I've renamed `SCPArgs` to `CommonArgs` now.